tsconfig.base.json

```diff
@@ -24,15 +24,15 @@
     "useUnknownInCatchVariables": false,
     "baseUrl": ".",
     "paths": {
-      "@fxa/payments/paypal": ["libs/payments/paypal/src/index.ts"],
+      "@fxa/payments/paypal": ["libs/payments/paypal/src/index"],
```

Comment:
During `fxa-graphql-api`'s `build`, it resolves the path alias to this value, but after the build it's compiled the library to javascript, so the `index.ts` doesn't exist anymore, but `index.js` does. So without this change you'd be getting unknown module.

Reply:
Thinking about it, I don't believe that script will work, since the issue is happening during the build.

As an alternative, `fxa-graphql-api/tsconfig.build.json` seems like a good place to make this change to the paths?